app/assets/stylesheets/decidim/_variables.scss

```diff
@@ -67,10 +67,7 @@ $twitter: #55acee !default;
 $facebook: #3b5998 !default;
 $google: #dd4b39 !default;
 
-$hel-grotesk: "HelsinkiGrotesk", Arial, -apple-system, BlinkMacSystemFont,
-  "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell",
-  "Fira Sans", "Droid Sans", "Helvetica Neue",
-  sans-serif;
+$main-font-face: "Source Sans Pro", sans-serif;
```

Comment:
ℹ️ I swapped it to Source Sans Pro everywhere. That's what Josh wanted. (I hope there's no awful side-effects anywhere.)